The job **`x86_64-gnu-distcheck`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/22275415511/job/64436648066) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/64436648066) [(plain)](https://github.com/rust-lang/rust/commit/77796608f152434cffef62643b028cc904c89075/checks/64436648066/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
REPOSITORY                                   TAG       IMAGE ID       CREATED       SIZE
ghcr.io/dependabot/dependabot-updater-core   latest    b72a662c47e3   3 weeks ago   790MB
=> Removing docker images...
Deleted Images:
untagged: ghcr.io/dependabot/dependabot-updater-core:latest
untagged: ghcr.io/dependabot/dependabot-updater-core@sha256:57ef9cc45f72cc4258ee1baa8243bc3cd55c0a0e570b6768c37346247be35f0d
deleted: sha256:b72a662c47e31df2e7bf59368b2b83be239f02a1baa721393717711a1a719df9
deleted: sha256:3e13ccd80f19769f39008cfc6549938e1ea4905f47b028c1df2dd6085191386c
deleted: sha256:842807995a512b2c5a9b241a3aecdbe79af6b0642d96fa5460cfcf0c9d8be295
deleted: sha256:0f9074b9f46f4570eb7cb4b65fcb3c3d909f9b1d14ca66b30508117b6deda303
deleted: sha256:2ca99cb9251d19157c56b5d91c8961bb4b35196a5ca9b4ffdccbf24abbfe2a5f
---
fmt error: Not running formatting checks; rustfmt.toml does not exist.
fmt error: This may happen in distributed tarballs.
tidy check
fatal: not a git repository (or any of the parent directories): .git
tidy [CI history]: failed to retrieve base commit: command did not execute successfully: "git" "rev-list" "--author-date-order" "--author=bors@rust-lang\\.org" "-n1" "HEAD" "--author" "122020455+rust-bors\\[bot\\]@users\\.noreply\\.github\\.com"
expected success, got: exit status: 128
. Some checks will be skipped.
tidy [gcc_submodule]: Cannot figure out the SHA of the GCC submodule
tidy: All tidy checks succeeded
x.py completions check
---
[183/3898] Building CXX object lib/Support/CMakeFiles/LLVMSupport.dir/zOSLibFunctions.cpp.o
[184/3898] Building CXX object lib/Support/CMakeFiles/LLVMSupport.dir/Signals.cpp.o
[185/3898] Building C object lib/Support/BLAKE3/CMakeFiles/LLVMSupportBlake3.dir/blake3_dispatch.c.o
[186/3898] Building C object lib/Support/BLAKE3/CMakeFiles/LLVMSupportBlake3.dir/blake3.c.o
[187/3898] Building CXX object lib/Support/LSP/CMakeFiles/LLVMSupportLSP.dir/Protocol.cpp.o
[188/3898] Building CXX object lib/Support/LSP/CMakeFiles/LLVMSupportLSP.dir/Transport.cpp.o
[189/3898] Linking CXX static library lib/libLLVMSupport.a
[190/3898] Building CXX object lib/Support/LSP/CMakeFiles/LLVMSupportLSP.dir/Logging.cpp.o
[191/3898] Linking CXX static library lib/libLLVMSupportLSP.a
[192/3898] Building CXX object lib/TableGen/CMakeFiles/LLVMTableGen.dir/Error.cpp.o
[193/3898] Building CXX object lib/TableGen/CMakeFiles/LLVMTableGen.dir/DetailedRecordsBackend.cpp.o
[194/3898] Building CXX object lib/TableGen/CMakeFiles/LLVMTableGen.dir/JSONBackend.cpp.o
[195/3898] Building CXX object lib/TableGen/CMakeFiles/LLVMTableGen.dir/Main.cpp.o
[196/3898] Building CXX object lib/TableGen/CMakeFiles/LLVMTableGen.dir/Parser.cpp.o
---
[229/3898] Building GenVT.inc...
[230/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/BuiltinCAS.cpp.o
[231/3898] Building Attributes.inc...
[232/3898] Building RuntimeLibcalls.inc...
[233/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/BuiltinUnifiedCASDatabases.cpp.o
[234/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/DatabaseFile.cpp.o
[235/3898] Building IntrinsicEnums.inc...
[236/3898] Building IntrinsicsAArch64.h...
[237/3898] Building IntrinsicsAMDGPU.h...
[238/3898] Building IntrinsicImpl.inc...
[239/3898] Building IntrinsicsLoongArch.h...
---
[294/3898] Building RISCVTargetParserDef.inc...
[295/3898] Building CXX object utils/TableGen/CMakeFiles/llvm-tblgen.dir/PseudoLoweringEmitter.cpp.o
[296/3898] Building CXX object utils/TableGen/CMakeFiles/llvm-tblgen.dir/RegisterBankEmitter.cpp.o
[297/3898] Building CXX object utils/TableGen/CMakeFiles/llvm-tblgen.dir/RegisterInfoEmitter.cpp.o
[298/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskCAS.cpp.o
[299/3898] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/BranchFolding.cpp.o
[300/3898] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/BranchRelaxation.cpp.o
[301/3898] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/BreakFalseDeps.cpp.o
[302/3898] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/BasicBlockSections.cpp.o
[303/3898] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/BasicBlockPathCloning.cpp.o
---
[428/3898] Building CXX object lib/InterfaceStub/CMakeFiles/LLVMInterfaceStub.dir/IFSHandler.cpp.o
[429/3898] Building CXX object lib/InterfaceStub/CMakeFiles/LLVMInterfaceStub.dir/ELFObjHandler.cpp.o
[430/3898] Building CXX object lib/IRPrinter/CMakeFiles/LLVMIRPrinter.dir/IRPrintingPasses.cpp.o
[431/3898] Building CXX object lib/IRReader/CMakeFiles/LLVMIRReader.dir/IRReader.cpp.o
[432/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskCommon.cpp.o
[433/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskDataAllocator.cpp.o
[434/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskGraphDB.cpp.o
[435/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/UnifiedOnDiskCache.cpp.o
[436/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskKeyValueDB.cpp.o
[437/3898] Building CXX object lib/CAS/CMakeFiles/LLVMCAS.dir/OnDiskTrieRawHashMap.cpp.o
[438/3898] Linking CXX static library lib/libLLVMCAS.a
[439/3898] Building CXX object lib/CGData/CMakeFiles/LLVMCGData.dir/CodeGenDataReader.cpp.o
[440/3898] Building CXX object lib/CGData/CMakeFiles/LLVMCGData.dir/CodeGenData.cpp.o
[441/3898] Building CXX object lib/CGData/CMakeFiles/LLVMCGData.dir/CodeGenDataWriter.cpp.o
[442/3898] Building CXX object lib/CGData/CMakeFiles/LLVMCGData.dir/OutlinedHashTree.cpp.o
[443/3898] Building CXX object lib/CGData/CMakeFiles/LLVMCGData.dir/OutlinedHashTreeRecord.cpp.o
---
[1285/3898] Building CXX object lib/Analysis/CMakeFiles/LLVMAnalysis.dir/ValueLattice.cpp.o
[1286/3898] Building CXX object lib/Analysis/CMakeFiles/LLVMAnalysis.dir/ValueLatticeUtils.cpp.o
[1287/3898] Building CXX object lib/Analysis/CMakeFiles/LLVMAnalysis.dir/ValueTracking.cpp.o
[1288/3898] Building CXX object lib/Analysis/CMakeFiles/LLVMAnalysis.dir/VectorUtils.cpp.o
[1289/3898] Building CXX object lib/DTLTO/CMakeFiles/LLVMDTLTO.dir/DTLTO.cpp.o
[1290/3898] Building CXX object lib/LTO/CMakeFiles/LLVMLTO.dir/LTO.cpp.o
[1291/3898] Building CXX object lib/LTO/CMakeFiles/LLVMLTO.dir/LTOModule.cpp.o
[1292/3898] Building CXX object lib/LTO/CMakeFiles/LLVMLTO.dir/LTOBackend.cpp.o
[1293/3898] Building CXX object lib/LTO/CMakeFiles/LLVMLTO.dir/LTOCodeGenerator.cpp.o
[1294/3898] Building CXX object lib/LTO/CMakeFiles/LLVMLTO.dir/UpdateCompilerUsed.cpp.o
---
[2142/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUMacroFusion.cpp.o
[2143/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUMCInstLower.cpp.o
[2144/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUMCResourceInfo.cpp.o
[2145/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUIGroupLP.cpp.o
[2146/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPULowerVGPREncoding.cpp.o
[2147/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUMarkLastScratchLoad.cpp.o
[2148/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUMIRFormatter.cpp.o
[2149/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUPerfHintAnalysis.cpp.o
[2150/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUPostLegalizerCombiner.cpp.o
[2151/3898] Building CXX object lib/Target/AMDGPU/CMakeFiles/LLVMAMDGPUCodeGen.dir/AMDGPUPrintfRuntimeBinding.cpp.o
---
[2808/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVVectorMaskDAGMutation.cpp.o
[2809/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVVectorPeephole.cpp.o
[2810/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVVLOptimizer.cpp.o
[2811/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVVMV0Elimination.cpp.o
[2812/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVVSETVLIInfoAnalysis.cpp.o
[2813/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVZacasABIFix.cpp.o
[2814/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/RISCVZilsdOptimizer.cpp.o
[2815/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/GISel/RISCVCallLowering.cpp.o
[2816/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/GISel/RISCVLegalizerInfo.cpp.o
[2817/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/GISel/RISCVPostLegalizerCombiner.cpp.o
[2818/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/GISel/RISCVO0PreLegalizerCombiner.cpp.o
[2819/3898] Building CXX object lib/Target/RISCV/CMakeFiles/LLVMRISCVCodeGen.dir/GISel/RISCVPreLegalizerCombiner.cpp.o
---
[3602/3898] Building CXX object tools/llvm-exegesis/lib/RISCV/CMakeFiles/LLVMExegesisRISCV.dir/RISCVExegesisPreprocessing.cpp.o
[3603/3898] Linking CXX static library lib/libLLVMExegesisRISCV.a
[3604/3898] Building CXX object tools/llvm-jitlink/CMakeFiles/llvm-jitlink.dir/llvm-jitlink.cpp.o
[3605/3898] Building CXX object tools/llvm-isel-fuzzer/CMakeFiles/llvm-isel-fuzzer.dir/DummyISelFuzzer.cpp.o
[3606/3898] Building CXX object tools/llvm-ir2vec/CMakeFiles/llvm-ir2vec.dir/llvm-ir2vec.cpp.o
[3607/3898] Building CXX object tools/llvm-isel-fuzzer/CMakeFiles/llvm-isel-fuzzer.dir/llvm-isel-fuzzer.cpp.o
[3608/3898] Building CXX object tools/llvm-jitlink/CMakeFiles/llvm-jitlink.dir/llvm-jitlink-elf.cpp.o
[3609/3898] Building CXX object tools/llvm-jitlink/CMakeFiles/llvm-jitlink.dir/llvm-jitlink-coff.cpp.o
[3610/3898] Linking CXX executable bin/llvm-exegesis
[3611/3898] Building CXX object tools/llvm-itanium-demangle-fuzzer/CMakeFiles/llvm-itanium-demangle-fuzzer.dir/DummyDemanglerFuzzer.cpp.o
---
[3792/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceVirtualRegisters.cpp.o
[3793/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceRegisterMasks.cpp.o
[3794/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceRegisterDefs.cpp.o
[3795/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceRegisterUses.cpp.o
[3796/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceSinkDefsToUses.cpp.o
[3797/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceTargetFeaturesAttr.cpp.o
[3798/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/ReduceUsingSimplifyCFG.cpp.o
[3799/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/SimplifyInstructions.cpp.o
[3800/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/RunIRPasses.cpp.o
[3801/3898] Building CXX object tools/llvm-reduce/CMakeFiles/llvm-reduce.dir/deltas/StripDebugInfo.cpp.o
---
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DWARFLinker/DWARFLinkerBase.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DWARFLinker/DWARFFile.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DWARFLinker/Parallel
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DWARFLinker/Parallel/DWARFLinker.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DTLTO
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/DTLTO/DTLTO.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/TargetParser
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/TargetParser/X86TargetParser.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/TargetParser/Triple.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/TargetParser/ARMTargetParserCommon.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/TargetParser/SubtargetFeature.h
---
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/FuzzMutate/RandomIRBuilder.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/Plugins
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/Plugins/PassPlugin.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/BuiltinCASContext.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/CASReference.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/OnDiskKeyValueDB.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/OnDiskGraphDB.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/FileOffset.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/OnDiskTrieRawHashMap.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/ObjectStore.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/UnifiedOnDiskCache.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/BuiltinUnifiedCASDatabases.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/BuiltinObjectHasher.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/CASID.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/ActionCache.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/MappedFileRegionArena.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/CAS/OnDiskDataAllocator.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/ToolDrivers
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/ToolDrivers/llvm-lib
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/ToolDrivers/llvm-lib/LibDriver.h
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/ToolDrivers/llvm-dlltool
-- Installing: /tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/llvm/include/llvm/ToolDrivers/llvm-dlltool/DlltoolDriver.h
---
---- [ui] tests/rustdoc-ui/ice-bug-report-url.rs stdout ----
Saved the actual stderr to `/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/test/rustdoc-ui/ice-bug-report-url/ice-bug-report-url.stderr`
diff of stderr:

5    |          ^ expected one of `->`, `where`, or `{`
6 
7 
- 
+ thread 'rustc' ($TID) panicked at /rustc-dev/77796608f152434cffef62643b028cc904c89075/compiler/rustc_errors/src/lib.rs:1531:17:
9 aborting due to `-Z treat-err-as-bug=1`
10 stack backtrace:
11 

Note: some mismatched output was normalized before being compared
- thread 'rustc' (437518) panicked at /rustc-dev/77796608f152434cffef62643b028cc904c89075/compiler/rustc_errors/src/lib.rs:1531:17:
+ thread 'rustc' ($TID) panicked at /rustc-dev/77796608f152434cffef62643b028cc904c89075/compiler/rustc_errors/src/lib.rs:1531:17:


The actual stderr differed from the expected stderr
To update references, rerun the tests and pass the `--bless` flag
To only update this specific test, also pass `--test-args ice-bug-report-url.rs`

error: 1 errors occurred comparing output.
status: exit status: 101
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" "/tmp/distcheck/distcheck-rustc-src/tests/rustdoc-ui/ice-bug-report-url.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/tmp/distcheck/distcheck-rustc-src/vendor" "--sysroot" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(test,FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "-o" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/test/rustdoc-ui/ice-bug-report-url" "-A" "internal_features" "-A" "unused_parens" "-A" "unused_braces" "-Cdebuginfo=0" "-Ztreat-err-as-bug"
stdout: none
--- stderr -------------------------------
error: internal compiler error: expected one of `->`, `where`, or `{`, found `<eof>`
##[error]  --> /tmp/distcheck/distcheck-rustc-src/tests/rustdoc-ui/ice-bug-report-url.rs:12:10
   |
LL | fn wrong()
   |          ^ expected one of `->`, `where`, or `{`


thread 'rustc' (437518) panicked at /rustc-dev/77796608f152434cffef62643b028cc904c89075/compiler/rustc_errors/src/lib.rs:1531:17:
aborting due to `-Z treat-err-as-bug=1`
stack backtrace:
---
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

error: the compiler unexpectedly panicked. This is a bug

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-rustdoc&template=ice.md

note: please make sure that you have updated to the latest nightly

note: rustc 1.95.0-nightly (77796608f 2026-02-22) (built from a source tarball) running on x86_64-unknown-linux-gnu

note: compiler flags: -Z threads=1 -Z simulate-remapped-rust-src-base=/rustc/FAKE_PREFIX -Z translate-remapped-path-to-local-path=no -Z ignore-directory-in-diagnostics-source-blocks=/cargo -Z ignore-directory-in-diagnostics-source-blocks=/tmp/distcheck/distcheck-rustc-src/vendor -C codegen-units=1 -Z ui-testing -Z deduplicate-diagnostics=no -Z write-long-types-to-disk=no -C strip=debuginfo -C debuginfo=0 -Z treat-err-as-bug

query stack during panic:
end of query stack
------------------------------------------

---
To only update this specific test, also pass `--test-args track-diagnostics.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" "/tmp/distcheck/distcheck-rustc-src/tests/rustdoc-ui/track-diagnostics.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/tmp/distcheck/distcheck-rustc-src/vendor" "--sysroot" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(test,FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "-o" "/tmp/distcheck/distcheck-rustc-src/build/x86_64-unknown-linux-gnu/test/rustdoc-ui/track-diagnostics" "-A" "internal_features" "-A" "unused_parens" "-A" "unused_braces" "-Cdebuginfo=0" "-Z" "track-diagnostics"
stdout: none
--- stderr -------------------------------
error[E0308]: mismatched types
##[error]  --> /tmp/distcheck/distcheck-rustc-src/tests/rustdoc-ui/track-diagnostics.rs:10:18
   |
---
test result: FAILED. 357 passed; 2 failed; 5 ignored; 0 measured; 0 filtered out; finished in 32.12s

Some tests failed in compiletest suite=rustdoc-ui mode=ui host=x86_64-unknown-linux-gnu target=x86_64-unknown-linux-gnu
Build completed unsuccessfully in 1:22:45
make: *** [Makefile:49: check] Error 1
Command `make check [workdir=/tmp/distcheck/distcheck-rustc-src]` failed with exit code 2
Created at: src/bootstrap/src/core/build_steps/test.rs:3423:5
Executed at: src/bootstrap/src/core/build_steps/test.rs:3429:10

Command has failed. Rerun with -v to see more details.
Bootstrap failed while executing `test distcheck`
Build completed unsuccessfully in 1:30:28
  local time: Sun Feb 22 12:07:58 UTC 2026
  network time: Sun, 22 Feb 2026 12:07:59 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/src/bootstrap/defaults/bootstrap.dist.toml
+++ b/src/bootstrap/defaults/bootstrap.dist.toml
@@ -24,6 +24,8 @@ channel = "auto-detect"
 download-rustc = false
 # Build the llvm-bitcode-linker
 llvm-bitcode-linker = true
+# Required to make builds reproducible.
+remap-debuginfo = true
 
 [dist]
 # Use better compression when preparing tarballs.

--- a/src/bootstrap/src/core/build_steps/test.rs
+++ b/src/bootstrap/src/core/build_steps/test.rs
@@ -3416,6 +3416,8 @@ fn distcheck_plain_source_tarball(builder: &Builder<'_>, plain_src_dir: &Path) {
     command("./configure")
         .arg("--set")
         .arg("rust.omit-git-hash=false")
+        .arg("--set")
+        .arg("rust.remap-debuginfo=false")
         .args(&configure_args)
         .arg("--enable-vendor")
         .current_dir(plain_src_dir)

--- a/src/ci/run.sh
+++ b/src/ci/run.sh
@@ -116,7 +116,6 @@ if [ "$DEPLOY$DEPLOY_ALT" = "1" ]; then
   else
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-llvm-static-stdcpp"
   fi
-  RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.remap-debuginfo"
 
   if [ "$DEPLOY_ALT" != "" ] && isLinux; then
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --debuginfo-level=2"
@@ -139,6 +138,8 @@ if [ "$DEPLOY$DEPLOY_ALT" = "1" ]; then
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-debug-assertions"
   fi
 else
+  RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.remap-debuginfo=false"
+
   # We almost always want debug assertions enabled, but sometimes this takes too
   # long for too little benefit, so we just turn them off.
   if [ "$NO_DEBUG_ASSERTIONS" = "" ]; then

